node/bundler.ts

```diff
@@ -121,7 +125,11 @@ const bundle = async (
 
   // Creating a final declarations array by combining the TOML file with the
   // deploy configuration API and the in-source configuration.
-  const declarations = getDeclarationsFromConfig(tomlDeclarations, functionsWithConfig, deployConfig)
+  const declarationsFromConfig = getDeclarationsFromConfig(tomlDeclarations, functionsWithConfig, deployConfig)
+
+  // If any declarations are autogenerated and are missing the generator field
+  // add a default string.
+  const declarations = addGeneratorFieldIfMissing(declarationsFromConfig, functions, internalFunctionsPath)
```

Comment:
nit: I don't see a reason why `addGeneratorFieldIfMissing` takes an array of declarations. I think it would read nicer if it took a single declaration and then the `.map` sat here. This would make it clear that you're just processing the same declarations array.

```ts
const declarations = declarationsFromConfig.map(declaration => addGeneratorFieldIfMissing(...
```

Reply:
Also, if `internalFunctionsPath` is `undefined` we probably shouldn't even call `addGeneratorFieldIfMissing`?

Reply:
I love this change, implementing!